public/data/food/processes.json

```diff
@@ -2148,25 +2149,26 @@
     "category": "transformation",
     "comment": "This process is a dummy process. It is assumed that no energy or water addition is needed for this process.",
     "impacts": {
-      "acd": 0.0,
-      "ozd": 0.0,
-      "cch": 0.0,
-      "fwe": 0.0,
-      "swe": 0.0,
-      "tre": 0.0,
-      "pco": 0.0,
-      "pma": 0.0,
-      "ior": 0.0,
-      "fru": 0.0,
-      "mru": 0.0,
-      "ldu": 0.0,
-      "wtu": 0.0,
-      "htc": 0.0,
-      "htn": 0.0,
-      "etf": 0.0,
-      "etf-c": 0.0,
-      "htc-c": 0.0,
-      "htn-c": 0.0
+      "bvi": 0,
+      "acd": 3.732661802e-21,
+      "ozd": 8.095288489e-26,
+      "cch": 3.059959849e-19,
+      "fwe": 4.402440369e-23,
```

Comment:
that's a bit strange as the process didn't change, the impact should be the same. I tried to run `make export_food` on the vm and I get 0 as before.
<img width="647" alt="Screenshot 2023-10-26 at 15 30 04" src="https://github.com/MTES-MCT/ecobalyse/assets/13175018/77645f30-583c-4529-ad1e-32216960c1cf">